msldap/examples/msldapclient.py

```diff
@@ -406,7 +408,7 @@ async def do_setsd(self, target_dn, sddl):
 			traceback.print_exc()
 			return False
 			
-	async def do_getsd(self, dn):
```

Comment:
Please do not extend this function to add your functionality, just create a new one with the name `do_getsdpp` or something similarly short and descriptive.  
The output of this function MUST be in the current format, as the same output can be used as an input to the `do_setsd` and it's guaranteed to reproduce the same SD (or at least very close to it)

Reply:
I left original functionality of `getsd` - if we use this command without parameters

Reply:
Whats about `getsd`? Should I separate this functional by `getsdp` and `getsdg`. Or we use tune behaviour via argument.
Currently It supports:
* `getsd DN`  - SDDL output
* `getsd DN g|grep|greppable`  - canonical greppable ACEs view (horizontal)
* `getsd DN p|pretty`  - canonical scrollable vertical view (vertical)